style.css

```diff
@@ -1079,3 +1078,154 @@ header {
   left: -35px;
   top: 90px;
 }
+
+.contact-img2 {
+  position: relative;
+  width: 231px;
+  height: 708px;
+  left: 176px;
+  top: -727px;
+}
+
+.contact-img {
+  position: relative;
+  width: 396px;
+  height: 781px;
+  left: 11px;
+  top: 11px;
+}
+
+.contact p {
+  position: relative;
+  width: 327px;
+  height: 102px;
+  left: 48px;
+  top: -777px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #ebebff;
+}
+
+.contact h2 {
+  position: relative;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  margin: 0;
+  margin-top: -582px;
+  top: -802px;
+  left: 89px;
+  color: #fff;
+}
+
+.contact {
+  position: relative;
+  width: 480px;
+  height: 920px;
+  left: 0;
+  top: 1600px;
+}
```

Comment:
- [ ] Please use flexbox to position your form elements as it is a better approach.